lib/git-fastclone.rb

```diff
@@ -371,9 +369,9 @@ def retriable_error?(error)
       error.to_s =~ /^STDERR:\n.*^#{Regexp.union(error_strings)}/m
     end
 
-    def print_formatted_error(error)
+    def print_formatted_error(error, location)
       indented_error = error.to_s.split("\n").map { |s| ">  #{s}\n" }.join
-      puts "Encountered a retriable error:\n#{indented_error}\n\nRemoving the fastclone cache."
+      puts "[INFO] Encountered a retriable error:\n#{indented_error}\n\n[WARN] Removing the fastclone cache at #{location}"
```

Comment:
NIT: I wonder if we should remove the latter half of this message and only print that from within `clear_cache` method.